### PR TITLE
Feature/alert component

### LINKED ISSUE #6: Create Notification component on FarmBlocks (modelsheet 2.0)
From the Zeplin url:

screen shot 2017-09-24 at 6 13 31 pm

### PR DESCRIPTION
**Pull request by fczuardi**
I am opening this PR against the monorepo branch, so the expected order of PR merges ideally would be:

1. merge this into monorepo branch
2. adapt button / tooltip components eventually to be part of the monorepo (to be filed)
3. merge monorepo with master

Regardless of when we merge monorepo branch with master, the monorepo branch is capable of publishing packages to npm, so we can keep using 2 parallel branches for some time.

Closes #6

### CODE REVIEW

**Comment by alcferreira** [comment]
We have commits that can be squashed into only one. Let's avoid "duplicated" commits

e.g.: `docs(authors): added Vitor` and `docs(authors): credit henlz` could be squashed into this one `feat(npm): initial package boilerplate`

It's the same for the `babel` related commits etc.

File: packages/alert/src/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+//@flow
+import type { StatelessFunctionalComponent, Node } from "react";
+import type { AlertType } from "./AlertTypes";
```
Comment (alcferreira) [inline packages/alert/src/index.js]: Why are we using these `import type`?

File: packages/alert/src/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import React from "react";
+import styled from "styled-components";
+import AlertTypes from "./AlertTypes";
+
+type AlertComponent = StatelessFunctionalComponent<{
```
Comment (alcferreira) [inline packages/alert/src/index.js]: Let's simplify this component...
Take a look at this [stateless component](https://flow.org/en/docs/react/components/#toc-stateless-functional-components) and this [default props](https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components). It's so much easier to read (:

File: packages/alert/src/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  const props = { text, dismissButton, type };
+  return <ThemedBaseAlert {...props} />;
+};
+
+type BaseAlertComponent = StatelessFunctionalComponent<{
+  text: string,
+  dismissButton: Node,
+  className: string
+}>;
+const BaseAlert: BaseAlertComponent = ({ text, dismissButton, className }) => {
```
Comment (alcferreira) [inline packages/alert/src/index.js]: I think we should have only one component per file.
So... Move this component to its own file.

File: packages/alert/src/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      {dismissButton}
+    </div>
+  );
+};
+
+type DismissButtonComponent = StatelessFunctionalComponent<{
+  onClick: Function,
+  className: string
+}>;
+const DismissButton: DismissButtonComponent = ({ onClick, className }) => {
```
Comment (alcferreira) [inline packages/alert/src/index.js]: Move this component to its own file.

**Comment by fczuardi** [comment]
Re: https://github.com/CraveFood/farmblocks/pull/20#issuecomment-332622277

I squashed a lot of things and brought down the pr from 41 to 18 commits

**Pull request merged**